refinery/ui/source/js/commons/ctrls/sharing-help-popover-details-ctrl.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Sharing Help Popover Details Ctrl
+ * @namespace SharingHelpPopoverDetailsCtrl
+ * @desc Controller for the sharing help popover details
+ * @memberOf refineryApp
+ */
+(function () {
+  'use strict';
+
+  angular
+    .module('refineryApp')
+    .controller('SharingHelpPopoverDetailsCtrl', SharingHelpPopoverDetailsCtrl);
+
+  SharingHelpPopoverDetailsCtrl.$inject = [
+    '$location'
+  ];
+
+
+  function SharingHelpPopoverDetailsCtrl (
+    $location
+  ) {
+    var vm = this;
+    vm.isCollaborationPage = false;
+
+   /*
+   * ---------------------------------------------------------
+   * LifeCycle Hooks
+   * ---------------------------------------------------------
+   */
+    vm.$onInit = function () {
+      // check if the user is on the collaboration page
+      vm.isCollaborationPage = $location.absUrl().indexOf('collaboration') > -1;
```

Comment:
Would it be worth while to be more precise here? `/collaboration/` should begin the path?

Reply:
I wouldn't imagine so, since the popover is just use on the home and collaboration page. But, I can add it.